Client/Frontend/Settings/Main/SettingDisclosureUtility.swift

```diff
@@ -8,10 +8,12 @@ import Shared
 
 struct SettingDisclosureUtility {
     static func buildDisclosureIndicator(theme: Theme) -> UIImageView {
-        let disclosureIndicator = UIImageView()
-        disclosureIndicator.image = UIImage(named: StandardImageIdentifiers.Large.chevronRight)?.withRenderingMode(.alwaysTemplate).imageFlippedForRightToLeftLayoutDirection()
-        disclosureIndicator.tintColor = theme.colors.actionSecondary
-        disclosureIndicator.sizeToFit()
-        return disclosureIndicator
+        // Ecosia: Custom chevron disclosure indicator
+        return ecosiaDisclosureIndicator
+//        let disclosureIndicator = UIImageView()
+//        disclosureIndicator.image = UIImage(named: StandardImageIdentifiers.Large.chevronRight)?.withRenderingMode(.alwaysTemplate).imageFlippedForRightToLeftLayoutDirection()
+//        disclosureIndicator.tintColor = theme.colors.actionSecondary
+//        disclosureIndicator.sizeToFit()
+//        return disclosureIndicator
```

Comment:
Thanks for catching and following the best practices when it comes to making comments so far 👏 
In this scenario, I believe when comparing the code with FF one we will create roughly 5 more conflict lines on top of the single commented ones.
In order to minimise that as much as possible, what I've been doing so far was mainly the following (better than explaining)

```
        /* Ecosia: Custom chevron disclosure indicator
        let disclosureIndicator = UIImageView()
        disclosureIndicator.image = UIImage(named: StandardImageIdentifiers.Large.chevronRight)?.withRenderingMode(.alwaysTemplate).imageFlippedForRightToLeftLayoutDirection()
        disclosureIndicator.tintColor = theme.colors.actionSecondary
        disclosureIndicator.sizeToFit()
        return disclosureIndicator
         */
        ecosiaDisclosureIndicator
```
so that the conflict lines will be 3 only.

WDYT?

Reply:
Looks better indeed! Will update it!

Wish you a great flight ❤️ and thank you for looking at it in the airport even 😮